src/utils/ghostTable.js

```diff
@@ -160,7 +160,19 @@ class GhostTable {
       this.injectTable();
     }
     arrayEach(this.columns, (column) => {
-      callback(column.col, outerWidth(column.table));
+      // The reason why `preciseOuterWidth` is here instead of just
+      // `outerWidth` is because on Safari, for whatever reason, using just
+      // `outerWidth` does not give enough pixels for all the text to render.
+      // `outerWidth` internally uses `offsetWidth`, which is always rounded to
+      // the nearest integer, whereas `preciseOuterWidth` uses
+      // `getBoundingClientRect`, calculates the current `scaleX` transform
+      // value, and rounds it up, always giving enough space for all the text to render on each line properly.
+      //
+      // (`getBoundingClientRect` on its own returns the width already
+      // rendered on the screen, which is usually useless when we want to reuse
+      // that measurement in some sibling/child element size, hence the need
+      // for `scaleX` calculation)
+      callback(column.col, preciseOuterWidth(column.table));
```

Comment:
Maybe use multi-line comment :)? Up to you. However, please try to do comment more compact. Text within line may be wider.

Reply:
Fair, I have my formatter set to 80 chars line, I'll try to do 120 next time. I followed the convention in this file, non jsdoc comments use single line comments. https://github.com/handsontable/handsontable/pull/7496/commits/899c1581faabb7d99032ed708756156f680375e4

Reply:
> I followed the convention in this file, non jsdoc comments use single line comments.

It's not about JSDoc for me, just the comments was quite short. There are 9 lines. But I won't push you to change it.